hash.go

```diff
@@ -103,7 +104,7 @@ func (s *solver) writeHashingInputs(w io.Writer) {
 	writeString(hhAnalyzer)
 	an, av := s.b.AnalyzerInfo()
 	writeString(an)
-	writeString(av.String())
+	writeString(fmt.Sprintf("%v", av))
```

Comment:
There's a decent chance it's the same thing under the hood, but the preferred approach here would be `strconv.Itoa()`.